_posts/2019-03-01-resizing-persistent-volumes.md

```diff
@@ -5,6 +5,10 @@ date: 2019-02-28
 author: jakub_scholz
 ---
 
+_Update (8th July 2019): 
+Strimzi now supports resizing of persistent volumes directly in the Strimzi Cluster Operator.
```

Comment:
Maybe you should reword it as well? This makes it sound like you have more options. But not that this blog post should not be followed any more. So maybe you can add some warning such as `Do not follow this blog post if you use Strimzi version newer than X (0.12?)`,

Reply:
I made the warning much stronger and pointed to the docs.